test/verify/check-networking-bridge

```diff
@@ -57,8 +57,20 @@ class TestBridge(NetworkCase):
         if "ubuntu" not in m.image and "debian" not in m.image and m.image not in ["fedora-coreos", "fedora-33"]:
             m.execute("! test -d /etc/sysconfig || test -f /etc/sysconfig/network-scripts/ifcfg-tbridge")
 
-        # Delete the bridge
+        # Check that the members are displayed and both On
         b.click("#networking-interfaces tr[data-interface='tbridge'] td:first-child")
+        b.wait_visible("#network-interface")
+        b.wait_present("#network-interface-members tr[data-interface='%s']" % iface1)
```

Comment:
The two wait_present() calls are redundant. Even if they weren't, they should go into wait_onoff() direclty, but wait_present() doesn't need to wait for parent selectors.

Reply:
Ok! (This was copy pasta, but I'll change it here.)